Add call out on reference-style links (#147) (#164)
carpentries-incubator/jekyll-pages-novice: _episodes/markdown.md

> {: .solution }
{: .challenge }

> ## Reference-style links
> 
> Up to now, we have used `inline style links` for example:
>
> ~~~
> [Carpentries Webpage](https://carpentries.org/)
> ~~~
> {: .language-markdown }
>
> If you use a link more than once, consider using so called `reference-style links` instead.
> Unlike inline-style links which have the URL inline with the description text, reference-style links

> > reference the URL via a reference.

Reviewer: We might be using "reference" too much in this sentence?
```suggestion
> reference the URL via a label.
```
"Label" is used [here](https://www.markdownguide.org/basic-syntax/#reference-style-links) which seems like a good decription.  Maybe we should link this page somewhere in this call out?

> > The reference goes into square brackets `[ ]`, for example:

Reviewer: If using label above, should use label here.

```suggestion
> The label goes into square brackets `[ ]`, for example:
```

> > The reference goes into square brackets `[ ]`, for example:
> > ~~~
> > [Carpentries Webpage][carpentries]
> >
> > [carpentries]: https://carpentries.org/
> > ~~~

Reviewer: To match the level of description you have elsewhere in this callout, I think it might be good to add more text above or below the code block about the 2nd half of the reference style link  (line 161). The below suggestion uses "label" instead of reference so if you decide not to keep this you might need to change it.

Something like...
```suggestion
> The label goes into square brackets `[ ]`, for example:
> ~~~
> [Carpentries Webpage][carpentries]
>
> [carpentries]: https://carpentries.org/
> ~~~
and then later, usually at the bottom of the page, you can add that label and the url it references to complete the link (as shown in the second line above).
```
